docs/modules/ROOT/pages/how-tos/exoscale/install.adoc

```diff
@@ -109,6 +109,7 @@ virt-edit -a rhcos-${RHCOS_VERSION}.qcow2 \
 exo storage upload rhcos-${RHCOS_VERSION}.qcow2 "sos://${CLUSTER_ID}-bootstrap" --acl public-read
```

Comment:
Did you check that this works without specifying the zone. I.e. is the name unique over all zone?

Reply:
I does work. I guess the name is unique but i have not tested the uniqueness especially...

Reply:
It is unique:
```sh
❯ exo storage create "sos://sebastian-widmer-test" --zone ch-gva-2
ok
❯ exo storage create "sos://sebastian-widmer-test" --zone ch-dk-2 
error: unable to create bucket: operation error S3: CreateBucket, https response error StatusCode: 409, BucketAlreadyOwnedByYou: 
❯ exo storage create "sos://sebastian-widmer-test-dk" --zone ch-dk-2
ok
```